documentation/tutorials/get-started-with-ash-oban.md

```diff
@@ -25,7 +25,7 @@ end
 
 ## Setup
 
-First, follow the oban setup guide.
+First, follow the [Oban setup guide](https://hexdocs.pm/oban/installation.html).
```

Comment:
Cool trick - hexdocs supports magical linking direct to another package's "extra"
```
[Oban setup guide](`e:oban:installation.html`)
```

Reply:
ooooo that is a very neat trick